src/controllers/docs.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { NextFunction, Request, Response, Router } from 'express';
+
+import redoc from 'redoc-express';
+
+import { Controller } from '../interfaces';
+
+export class DocsController implements Controller {
+
+  public basepath = '/docs';
+
+  public boot(): Router {
+    const router = Router();
+    
+    router.get('/docs/openapi.yaml', (req, res) => {
+      res.sendFile('.../openapi.yaml');
+    });
```

Comment:
when I tested this branch this was returning a `500` error. We could change to this:
```suggestion
    router.get(`${this.basepath}/openapi.yaml`, (_, res) => {
      res.sendFile('openapi.yaml', { root: '.' });
    });
```
wdyt?